roachpb/data_test.go

```diff
@@ -415,6 +415,9 @@ func TestTransactionString(t *testing.T) {
 	if str := txn.String(); str != expStr {
 		t.Errorf("expected txn %s; got %s", expStr, str)
 	}
+
+	var txnEmpty Transaction
+	_ = txnEmpty.String() // prevent regression of NPE
```

Comment:
Shouldn't this be `txnEmpty.Short()`? Or is there some hidden way that this is testing the NPE that I'm not seeing?